lesson_10/solid/solid_app/src/test/java/com/codedifferently/lesson10/library/LibraryTest.java

```diff
@@ -1,13 +1,18 @@
-package com.codedifferently.lesson10.library;
+/*package com.codedifferently.lesson10.library;
```

Comment:
Have to actually run the tests for the code to be accepted. Try working on your tests one at a time instead of attempting to make several changes at once.